# Meskó Balázs <meskobalazs@gmail.com>, 2016. #zanata
# Meskó Balázs <meskobalazs@gmail.com>, 2018. #zanata
# Meskó Balázs <meskobalazs@gmail.com>, 2019. #zanata
# Balázs Meskó <meskobalazs@mailbox.org>, 2020, 2021.
# Anonymous <noreply@weblate.org>, 2020.
msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2021-04-05 09:18-0400\n"
"PO-Revision-Date: 2021-02-10 13:41+0000\n"
"Last-Translator: Balázs Meskó <meskobalazs@mailbox.org>\n"
"Language-Team: Hungarian <https://translate.fedoraproject.org/projects/dnf/"
"dnf-plugins-extras/hu/>\n"
"Language: hu\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=n != 1;\n"
"X-Generator: Weblate 4.4.2\n"

#: plugins/kickstart.py:57
msgid "Install packages defined in a kickstart file on your system"
msgstr "A kickstart fájlban megadott csomagok telepítése a rendszerre"

#: plugins/kickstart.py:62
msgid "kickstart file"
msgstr "kickstart fájl"

#: plugins/kickstart.py:80
#, python-format
msgid "file cannot be parsed: %s"
msgstr "a fájl nem dolgozható fel: %s"

#: plugins/rpm_conf.py:65
msgid "Adding '{}' to list of handling packages for rpmconf"
msgstr "A(z) „{}” hozzáadása a rpmconf kezelőcsomagjaihoz"

#: plugins/rpm_conf.py:71
msgid "rpmconf plugin will not run in non-interactive mode"
msgstr "az rpmconf bővítmény nem fog futni nem interaktív módban"

#: plugins/rpm_conf.py:84
msgid "ignoring sys.exit from rpmconf due to missing MERGE variable"
msgstr "a sys.exit mellőzése az rpmconfban a hiányzó MERGE változó miatt"

#: plugins/rpm_conf.py:88
msgid "ignoring sys.exit from rpmconf due to missing file"
msgstr "a sys.exit mellőzése az rpmconfban egy hiányzó fájl miatt"

#: plugins/showvars.py:46
msgid "show all active dnf variables"
msgstr "az összes aktív dnf változó megjelenítése"

#: plugins/snapper.py:49
#, python-format
msgid "connect to snapperd failed: %s"
msgstr "a snapperdhez kapcsolódás sikertelen: %s"

#: plugins/snapper.py:55
msgid "creating pre_snapshot"
msgstr "művelet előtti pillanatfelvétel készítése"

#: plugins/snapper.py:61
#, python-format
msgid "created pre_snapshot %d"
msgstr "%d. művelet előtti pillanatfelvétel elkészült"

#: plugins/snapper.py:65
#, python-format
msgid "creating pre_snapshot failed: %s"
msgstr "A művelet előtti pillanatfelvétel készítése meghiúsult: %s"

#: plugins/snapper.py:74
msgid "skipping post_snapshot because creation of pre_snapshot failed"
msgstr ""
"művelet utáni pillanatfelvétel kihagyása, mert az előtti készítése meghiúsult"

#: plugins/snapper.py:80
msgid "creating post_snapshot"
msgstr "művelet utáni pillanatfelvétel készítése"

#: plugins/snapper.py:85
#, python-format
msgid "created post_snapshot %d"
msgstr "%d. művelet utáni pillanatfelvétel elkészült"

#: plugins/snapper.py:89
#, python-format
msgid "creating post_snapshot failed: %s"
msgstr "A művelet utáni pillanatfelvétel készítése meghiúsult: %s"

#. Translators: This string is only used in unit tests.
#: plugins/system_upgrade.py:46
msgid "the color of the sky"
msgstr "az égbolt színe"

#: plugins/system_upgrade.py:60
msgid "Need a --releasever greater than the current system version."
msgstr ""
"A jelenlegi rendszer verziójánál nagyobb --releasever értéket kell megadnia."

#. Translators: do not change "reboot" here
#: plugins/system_upgrade.py:62
#, python-brace-format
msgid ""
"Download complete! Use 'dnf {command} reboot' to start the upgrade.\n"
"To remove cached metadata and transaction use 'dnf {command} clean'"
msgstr ""
"Letöltés kész! Használja a „dnf {command} reboot” parancsot a frissítés "
"indításához.\n"
"A tárolt metaadatok és a tranzakció eltávolításához használja ezt: „dnf "
"{command} clean”"

#: plugins/system_upgrade.py:65
msgid "Sorry, you need to use 'download --releasever' instead of '--network'"
msgstr ""
"Sajnáljuk, a „download --releasever” parancsot kell használni a „--network” "
"helyett"

#: plugins/system_upgrade.py:74
msgid "Reboot turned off, not rebooting."
msgstr ""

#: plugins/system_upgrade.py:123
#, python-format
msgid "Screen blanking can't be disabled: %s"
msgstr "A képernyő törlése nem tiltható le: %s"

#: plugins/system_upgrade.py:143
#, python-format
msgid "Failed loading state file: %s, continuing with empty state."
msgstr "Állapotfájl betöltése meghiúsult: %s, folytatás üres állapottal."

#: plugins/system_upgrade.py:289
msgid "The following boots appear to contain upgrade logs:"
msgstr "A következő indulások tartalmazzák a frissítési naplókat:"

#: plugins/system_upgrade.py:299
msgid "-- no logs were found --"
msgstr "-- nem található naplófájl --"

#: plugins/system_upgrade.py:314
msgid "Cannot find logs with this index."
msgstr "Ilyen index alatt nem található naplófájl."

#: plugins/system_upgrade.py:323
msgid "Unable to match systemd journal entry"
msgstr "Nincs találat a systemd naplóbejegyzésre"

#: plugins/system_upgrade.py:344
msgid "Prepare system for upgrade to a new release"
msgstr "Rendszer előkészítése az új verzióra történő frissítéshez"

#: plugins/system_upgrade.py:360
msgid "keep installed packages if the new release's version is older"
msgstr "megtartja a telepített csomagokat, ha az új kiadás verziója régebbi"

#: plugins/system_upgrade.py:364
msgid "which logs to show"
msgstr "megjelenítendő naplók"

#: plugins/system_upgrade.py:398
#, python-brace-format
msgid "Incompatible version of data. Rerun 'dnf {command} download [OPTIONS]'"
msgstr ""
"Nem kompatibilis adatverziók. Futtassa újra: „dnf {command} download "
"[KAPCSOLÓK]”"

#: plugins/system_upgrade.py:435
msgid "Command 'offline-distrosync' cannot be used with --no-downgrade option"
msgstr ""
"Az „offline-distrosync” parancs nem használható a --no-downgrade kapcsolóval"

#: plugins/system_upgrade.py:457
msgid "Additional information for System Upgrade: {}"
msgstr "További információk a rendszerfrissítéshez: {}"

#: plugins/system_upgrade.py:460
msgid ""
"Before you continue ensure that your system is fully upgraded by running "
"\"dnf --refresh upgrade\". Do you want to continue"
msgstr ""
"A folytatás előtt győződjön meg róla, hogy a rendszer teljesen frissítve "
"lett a „dnf --refresh upgrade” paranccsal. Folytatja"

#: plugins/system_upgrade.py:464
msgid "Operation aborted."
msgstr "Művelet megszakítva."

#: plugins/system_upgrade.py:522
msgid "system is not ready for upgrade"
msgstr "a rendszer nem áll készen a frissítésre"

#: plugins/system_upgrade.py:525
#, python-brace-format
msgid ""
"the transaction was not prepared for '{command}'. Rerun 'dnf {command} "
"download [OPTIONS]'"
msgstr ""
"a tranzakció nem készült el ehhez: „{command}”. Futtassa újra: „dnf "
"{command} download [KAPCSOLÓK]”"

#: plugins/system_upgrade.py:529
msgid "upgrade is already scheduled"
msgstr "a frissítés már ütemezve van"

#: plugins/system_upgrade.py:535
msgid "trigger file does not exist. exiting quietly."
msgstr "a kiváltó fájl nem létezik. csendes kilépés."

#: plugins/system_upgrade.py:538
msgid "another upgrade tool is running. exiting quietly."
msgstr "fut egy másik frissítési eszköz. csendes kilépés."

#: plugins/system_upgrade.py:547
#, python-brace-format
msgid "use 'dnf {command} reboot' to begin the upgrade"
msgstr "a telepítés folytatásához használja a „dnf {command} reboot” parancsot"

#: plugins/system_upgrade.py:565
msgid "Rebooting to perform upgrade."
msgstr "Újraindítás a frissítés végrehajtásához."

#: plugins/system_upgrade.py:598
msgid "Starting offline upgrade. This will take a while."
msgstr "Offline frissítés elkezdése. Ez eltart egy darabig."

#: plugins/system_upgrade.py:600
msgid "Starting offline distrosync. This will take a while."
msgstr "Offline disztribúciószinkronizálás elkezdése. Ez eltart egy darabig."

#: plugins/system_upgrade.py:602
msgid "Starting system upgrade. This will take a while."
msgstr "Rendszerfrissítés elkezdése. Ez eltart egy darabig."

#: plugins/system_upgrade.py:618
msgid "Cleaning up downloaded data..."
msgstr "Letöltött adatok tisztítása…"

#: plugins/system_upgrade.py:650
msgid "Transaction saved to {}."
msgstr "Tranzakció ide mentve: {}."

#: plugins/system_upgrade.py:653
msgid "Error storing transaction: {}"
msgstr "Hiba történt a tranzakció tárolásakor: {}"

#: plugins/system_upgrade.py:675
msgid "Download finished."
msgstr "Letöltés befejezve."

#: plugins/system_upgrade.py:678 plugins/system_upgrade.py:679
msgid "Upgrade complete! Cleaning up and rebooting..."
msgstr "Frissítés kész! Tisztítás és újraindítás…"

#: plugins/system_upgrade.py:688
msgid "Prepare offline upgrade of the system"
msgstr "A rendszer offline frissítésének előkészítése"

#: plugins/system_upgrade.py:693
msgid "Prepare offline distrosync of the system"
msgstr "A rendszer offline disztribúciófrissítésének előkészítése"

#: plugins/torproxy.py:31
msgid "Disabling torproxy plugin: cannot connect to the Tor network"
msgstr "A torproxy bővítmény letiltása: nem lehet a Tor hálózathoz kapcsolódni"

#: plugins/tracer.py:69
msgid "For more information run:"
msgstr "További információért futtassa:"

#: plugins/tracer.py:96
msgid "Call to Tracer API ended unexpectedly:"
msgstr "A Tracer API hívása váratlanul véget ért:"

#: plugins/tracer.py:98
msgid ""
"Please visit https://github.com/FrostyX/tracer/issues and submit the issue. "
"Thank you"
msgstr ""
"Keresse fel a https://github.com/FrostyX/tracer/issues oldalt, és küldje be "
"a problémát. Köszönjük"

#: plugins/tracer.py:100
msgid "We apologize for any inconvenience"
msgstr "Elnézést kérünk a kellemetlenségekért"

#~ msgid "Unable to match package: %s"
#~ msgstr "Nincs találat a csomagra: %s"

#~ msgid "Unable to match some of packages"
#~ msgstr "Nincs találat néhány csomagra"
